### PR TITLE
Feature/validation feedback

### PR DESCRIPTION
**Pull request by krebsni**

### CODE REVIEW

File: qhana_plugin_runner/templates/simple_template.html [hunk 1]
```diff
@@ -31,6 +31,9 @@
             {% if done %}
             {{ forms.submit("done", action=done)}}
             {% endif %}
+            {% if errors == {} and valid %}
```
Comment (buehlefs) [inline qhana_plugin_runner/templates/simple_template.html:34]:
```suggestion
            {% if not errors and valid %}
```

File: qhana_plugin_runner/templates/simple_template.html [hunk 2]
```diff
@@ -31,6 +31,9 @@
...
+            <mat-icon aria-hidden="true" class="material-icon notranslate material-icons ng-star-inserted qhana-valid-icon">done</mat-icon>
```
Comment (buehlefs) [inline qhana_plugin_runner/templates/simple_template.html:35]: This actually works? A better solution would be to inline the icon SVG. Also make sure that the valid indicator has at least some text that can be seen (and read by screenreaders) explaining what the green checkmark actually means.

**Review by buehlefs — CHANGES_REQUESTED** [review]
I've only left comments for the first occurrence but please fix them all.

File: plugins/pca.py [hunk 3]
```diff
@@ -0,0 +1,382 @@
+# Copyright 2021 QHAna plugin runner contributors.
```
[the remark this answers is not in the document]
Comment (krebsni) [reply plugins/pca.py]: Not sure must have somehow slipped in when pca was refactored into own folder... I just deleted it for now.

File: qhana_plugin_runner/templates/simple_template.html [hunk 4]
```diff
@@ -31,8 +31,9 @@
             {% if done %}
             {{ forms.submit("done", action=done)}}
             {% endif %}
-            {% if errors == {} and valid %}
-            <mat-icon aria-hidden="true" class="material-icon notranslate material-icons ng-star-inserted qhana-valid-icon">done</mat-icon>
+            {% if not errors and valid %}
+            <span class="qhana-input-description"> Validation successful!</span>
+            <svg class="checkmark" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><path class="checkmark__check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8"/></svg>
```
Comment (buehlefs) [inline qhana_plugin_runner/templates/simple_template.html:36]: Add source and license info in html comment

**Review by buehlefs — APPROVED** [review]

**Pull request merged**